_layouts/default.html

```diff
@@ -5,10 +5,7 @@
 
   <body>
     {% include header.html %}
-    {% if page.url == '/' %}{% include banner-1-0.html %}{% endif %}
```

Comment:
👍  on removing it.

Might still be worth it to rename it to `banner.html`, remove the copy and just comment out here. So that at some point in the future, when there is a need for a banner, it can quickly be added again.